src/Block/Service/AbstractCategoriesBlockService.php

```diff
@@ -117,11 +129,14 @@ public function buildEditForm(FormMapper $formMapper, BlockInterface $block): vo
     public function configureSettings(OptionsResolver $resolver): void
     {
         $resolver->setDefaults([
-            'title' => 'Categories',
+            'title' => null,
```

Comment:
It is normal to see this title to null?

Reply:
@core23 can you answer?

Reply:
if it is present on the PR fine by me, I was asking if the merge is correct only :)

Reply:
oh, it is present :+1: